### PR TITLE
Trace cmd

### PR DESCRIPTION
**Pull request by atheurer**
Add support trace-cmd and intel-sst

### CODE REVIEW

File: kerneltools-start [hunk 1]
```diff
@@ -129,6 +135,27 @@ for subtool in `echo $subtools | sed -e 's/,/ /g'`; do
...
+            /usr/local/bin/trace-cmd list -t
+            echo "Starting function trace with functions: $trace_cmd_functions"
+            cmd="/usr/local/bin/trace-cmd record"
+            cmd+=" -p function"
+            cmd+=" --func-stack"
+            cmd+=" -l smp_call_function_many"
+            cmd+=" -l smp_call_function_any"
+            cmd+=" -l smp_call_function_single_async"
+            cmd+=" -l smp_call_function_single"
+            cmd+=" -l smp_call_function_single_interrupt"
```
Comment (k-rister) [inline kerneltools-start:150]: Are you really wanting to hard code all this stuff?

File: workshop.json [hunk 2]
```diff
@@ -8,7 +8,7 @@
         {
             "name": "default",
             "requirements": [
-                "bzip2_src", "bison_src", "gettext_src", "flex_src", "tools_src"
+                "bzip2_src", "bison_src", "gettext_src", "flex_src", "tools_src", "libtraceevent_src", "libtracefs_src", "tracecmd_src", "bcc", "rt-trace-bpf"
```
Comment (k-rister) [inline workshop.json:11]: Is the bpf/bcc stuff relevant to this?

Comment (atheurer) [reply workshop.json:11]: probably not,  I will remove for a later PR

File: kerneltools-start (hunk 1 above)
Comment (atheurer) [reply kerneltools-start:150]: I thought all of this was removed with a more recent commit, maybe not.

**Pull request merged**